styles/main/style.css

```diff
@@ -403,6 +410,72 @@ html {
   justify-content: center;
 }
 
+/* Blogs section */
+.blogs {
+  margin-top: 140px;
+}
+.blogs__title {
+  text-align: center;
+  font-size: 54px;
+  font-weight: 500;
+}
```

Comment:
I'm pretty sure that blog shouldn't be plural. It's "blog" and the items inside should be "article" or "blog__article". Maybe even- the container - "articles" and each item "article".